dap/BESDapResponseBuilder.cc

```diff
@@ -1453,6 +1454,19 @@ void BESDapResponseBuilder::send_dap4_data(ostream &out, DMR &dmr, bool with_mim
     }
 }
 
+const auto DAP4_CHECKSUMS_KEY="dap4_checksums";
+
+bool use_dap4_checksums() {
+    bool found;
```

Comment:
Should provide an initial value for found.

Reply:
Thanks! Fixed.